packages/css-library/README.md

```diff
@@ -1,3 +1,17 @@
 # css-library
 
-This package is the early discovery work on basing VADS styles off of USWDS v3.
+## Stylesheets
+
+The stylesheets generated by this library are based on USWDS version 3 and are intended to eventually replace the Formation CSS library:
```

Comment:
```suggestion
The stylesheets generated by this library are based on [USWDS version 3](https://github.com/uswds/uswds) and are intended to eventually replace the Formation CSS library:
```

I always find it helpful to include links to external references.

Reply:
@micahchiang Good call! Updated.

My hope is that this readme continues to evolve as we are figuring out how it's integrating into vets-website. Like I'm not entirely clear at the moment how the CSS Library and the [minimal stylesheet](https://github.com/department-of-veterans-affairs/vets-website/blob/017fb54a869467bb1b569e331ccd3def64239596/src/platform/site-wide/sass/minimal.scss) are working together to "replace Formation". I've added what's here because that is what was in the [original CSS Library plan](https://vfs.atlassian.net/wiki/spaces/DST/pages/2181857291/Plan+-+Design+System+CSS+Utility+Class+Library+Rebuild).